src/extension/app/store/app.js

```diff
@@ -647,6 +647,7 @@ export class AppStore {
         'helpdismissed',
         'helpacknowlegded',
         'helpoptedout',
+        'projectadded',
```

Comment:
Wondering if we should rename this since it represents the event for both adding and removing a project (`projectToggled`?)

Reply:
Wouldn't it be interesting for us to know how many people add vs remove projects?